Auto update failing for vulkan-validation-layers

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [vulkan-validation-layers](https://github.com/termux/termux-packages/blob/503eceb26965337cc79a37b8b3cf76103b16726f/packages/vulkan-validation-layers) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating vulkan-validation-layers [Current version: 1.3.289]
INFO: package being updated to 1.3.290.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 13008    0 13008    0     0  52960      0 --:--:-- --:--:-- --:--:-- 52960
100 5142k    0 5142k    0     0  5389k      0 --:--:-- --:--:-- --:--:-- 7246k

You are about to commit these changes:

--------------------
diff --git a/packages/vulkan-validation-layers/build.sh b/packages/vulkan-validation-layers/build.sh
index 102c303ca..81e2ac2c9 100644
--- a/packages/vulkan-validation-layers/build.sh
+++ b/packages/vulkan-validation-layers/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/KhronosGroup/Vulkan-ValidationLayers
 TERMUX_PKG_DESCRIPTION="Vulkan Validation Layers"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.3.289"
+TERMUX_PKG_VERSION="1.3.290"
 TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=4054b035879e90d6f0b7f993069fa50499c87776ae6c981c65f8a13ed69e12a1
+TERMUX_PKG_SHA256=37fd611f1c5c5879a68c6f63b09e600a3858f519aa147a2535961fa3d8e889ae
 TERMUX_PKG_DEPENDS="libc++, vulkan-loader"
 TERMUX_PKG_BUILD_DEPENDS="libwayland, libx11, libxcb, libxrandr, spirv-headers, spirv-tools, vulkan-headers (=${TERMUX_PKG_VERSION}), vulkan-utility-libraries (=${TERMUX_PKG_VERSION})"
 TERMUX_PKG_ANTI_BUILD_DEPENDS="vulkan-loader"
--------------------

bump(main/vulkan-validation-layers): 1.3.290


Not committing to Git!

INFO: Trying to build package.
INFO: Package vulkan-validation-layers exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building vulkan-validation-layers for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0  82616      0 --:--:-- --:--:-- --:--:-- 83126
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   4931      0 --:--:-- --:--:-- --:--:--  4958
gpg: Signature made Fri 12 Jul 2024 09:10:24 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  87312      0 --:--:-- --:--:-- --:--:-- 87631
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   5213      0 --:--:-- --:--:-- --:--:--  5206
gpg: Signature made Fri 12 Jul 2024 04:25:49 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  83645      0 --:--:-- --:--:-- --:--:-- 84102
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   5098      0 --:--:-- --:--:-- --:--:--  5110
gpg: Signature made Fri 12 Jul 2024 08:15:25 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libc++@26b if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-support@29 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libexpat@2.6.2 if necessary...
Found libexpat in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libexpat to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libffi@3.4.6-1 if necessary...
Found libffi in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libffi to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libiconv@1.17 if necessary...
Found libiconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libiconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency iconv@1.17 if necessary...
Found iconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting iconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblzma@5.6.2 if necessary...
Found liblzma in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting liblzma to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xz-utils@5.6.2 if necessary...
Found xz-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xz-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.4.20231001-4 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.4.20231001-4 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.4.20231001-4 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.4.20231001-4 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.2.10 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gdbm@1.23 if necessary...
Found gdbm in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting gdbm to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-posix-semaphore@0.1-3 if necessary...
Found libandroid-posix-semaphore in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-posix-semaphore to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libbz2@1.0.8-6 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-6 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2024.07.02 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2024.07.02 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.2.1-1 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.2.1-1 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcrypt@0.2-5 if necessary...
Found libcrypt in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcrypt to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tcl@8.6.14 if necessary...
Found tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite@3.46.0 if necessary...
Found libsqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sqlite@3.46.0 if necessary...
Found sqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting sqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite-tcl@3.46.0 if necessary...
Found libsqlite-tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite-tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency brotli@1.1.0 if necessary...
Found brotli in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting brotli to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libpng@1.6.43 if necessary...
Found libpng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libpng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency freetype@2.13.2 if necessary...
Found freetype in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting freetype to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ttf-dejavu@2.37-8 if necessary...
Found ttf-dejavu in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ttf-dejavu to /home/builder/.termux-build/_cache-all...
Downloading dependency fontconfig@2.15.0 if necessary...
Found fontconfig in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fontconfig-utils@2.15.0 if necessary...
Found fontconfig-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xorg-util-macros@1.20.1 if necessary...
Found xorg-util-macros in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorg-util-macros to /home/builder/.termux-build/_cache-all...
Downloading dependency xorgproto@2024.1 if necessary...
Found xorgproto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorgproto to /home/builder/.termux-build/_cache-all...
Downloading dependency libxau@1.0.11 if necessary...
Found libxau in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxau to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxdmcp@1.1.5 if necessary...
Found libxdmcp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxdmcp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xcb-proto@1.17.0 if necessary...
Found xcb-proto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xcb-proto to /home/builder/.termux-build/_cache-all...
Downloading dependency python-xcbgen@1.17.0 if necessary...
Found python-xcbgen in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-xcbgen to /home/builder/.termux-build/_cache-all...
Downloading dependency libxcb@1.17.0 if necessary...
Found libxcb in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxcb to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xtrans@1.5.0 if necessary...
Found xtrans in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xtrans to /home/builder/.termux-build/_cache-all...
Downloading dependency libx11@1.8.9 if necessary...
Found libx11 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libx11 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxrender@0.9.11 if necessary...
Found libxrender in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxrender to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxft@2.3.8 if necessary...
Found libxft in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxft to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxext@1.3.6 if necessary...
Found libxext in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxext to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxss@1.2.4 if necessary...
Found libxss in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxss to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tk@8.6.14 if necessary...
Found tk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python@3.11.9-3 if necessary...
Found python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-tkinter@3.11.9-3 if necessary...
Found python-tkinter in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-tkinter to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-ensurepip-wheels@3.11.9-3 if necessary...
Found python-ensurepip-wheels in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-ensurepip-wheels to /home/builder/.termux-build/_cache-all...
Downloading dependency libxml2@2.13.2-1 if necessary...
Found libxml2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxml2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-utils@2.13.2-1 if necessary...
Found libxml2-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxml2-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-python@2.13.2-1 if necessary...
Found libxml2-python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxml2-python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-python-static@2.13.2-1 if necessary...
Found libxml2-python-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxml2-python-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwayland@1.22.0-1 if necessary...
Found libwayland in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libwayland to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwayland-cross-scanner@1.22.0-1 if necessary...
Found libwayland-cross-scanner in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libwayland-cross-scanner to /home/builder/.termux-build/_cache-all...
Downloading dependency libxrandr@1.5.4 if necessary...
Found libxrandr in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxrandr to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency spirv-headers@1.3.283.0 if necessary...
Found spirv-headers in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/s/spirv-headers/spirv-headers_1.3.283.0_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  112k  100  112k    0     0   183k      0 --:--:-- --:--:-- --:--:--  183k
extracting spirv-headers to /home/builder/.termux-build/_cache-all...
Downloading dependency spirv-tools@1.3.283.0 if necessary...
Found spirv-tools in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/s/spirv-tools/spirv-tools_1.3.283.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1950k  100 1950k    0     0  1961k      0 --:--:-- --:--:-- --:--:-- 1960k
extracting spirv-tools to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency vulkan-headers@1.3.290 if necessary...
Found vulkan-headers in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting vulkan-headers to /home/builder/.termux-build/_cache-all...
Downloading dependency vulkan-utility-libraries@1.3.290 if necessary...
Download of vulkan-utility-libraries@1.3.290 from https://packages-cf.termux.dev/apt/termux-main failed, building instead
termux - building vulkan-utility-libraries for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0   225k      0 --:--:-- --:--:-- --:--:--  229k
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0  13580      0 --:--:-- --:--:-- --:--:-- 13655
gpg: Signature made Fri 12 Jul 2024 09:10:24 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0   205k      0 --:--:-- --:--:-- --:--:--  206k
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0  14045      0 --:--:-- --:--:-- --:--:-- 14118
gpg: Signature made Fri 12 Jul 2024 04:25:49 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0   180k      0 --:--:-- --:--:-- --:--:--  183k
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0  11768      0 --:--:-- --:--:-- --:--:-- 11900
gpg: Signature made Fri 12 Jul 2024 08:15:25 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libc++@26b if necessary...
Skipping already built dependency libc++@26b
Downloading dependency vulkan-headers@1.3.290 if necessary...
Skipping already built dependency vulkan-headers@1.3.290
Applying patch: 0001-do-not-build-for-android.patch
-- The CXX compiler identification is Clang 17.0.2
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/builder/.termux-build/_cache/android-r26b-api-24-v4/bin/aarch64-linux-android-clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring done (0.2s)
-- Generating done (0.0s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_ANDROID_STANDALONE_TOOLCHAIN
    CMAKE_C_FLAGS
    CMAKE_FIND_ROOT_PATH_MODE_LIBRARY
    CMAKE_USE_SYSTEM_LIBRARIES
    DOXYGEN_EXECUTABLE


-- Build files have been written to: /home/builder/.termux-build/vulkan-utility-libraries/build
[1/12] Building CXX object src/layer/CMakeFiles/VulkanLayerSettings.dir/vk_layer_settings_helper.cpp.o
[2/12] Building CXX object src/layer/CMakeFiles/VulkanLayerSettings.dir/layer_settings_manager.cpp.o
[3/12] Building CXX object src/layer/CMakeFiles/VulkanLayerSettings.dir/vk_layer_settings.cpp.o
[4/12] Building CXX object src/layer/CMakeFiles/VulkanLayerSettings.dir/layer_settings_util.cpp.o
[5/12] Building CXX object src/vulkan/CMakeFiles/VulkanSafeStruct.dir/vk_safe_struct_ext.cpp.o
[6/12] Building CXX object src/vulkan/CMakeFiles/VulkanSafeStruct.dir/vk_safe_struct_khr.cpp.o
[7/12] Building CXX object src/vulkan/CMakeFiles/VulkanSafeStruct.dir/vk_safe_struct_core.cpp.o
[8/12] Linking CXX static library src/layer/libVulkanLayerSettings.a
[9/12] Building CXX object src/vulkan/CMakeFiles/VulkanSafeStruct.dir/vk_safe_struct_manual.cpp.o
[10/12] Building CXX object src/vulkan/CMakeFiles/VulkanSafeStruct.dir/vk_safe_struct_utils.cpp.o
[11/12] Building CXX object src/vulkan/CMakeFiles/VulkanSafeStruct.dir/vk_safe_struct_vendor.cpp.o
[12/12] Linking CXX static library src/vulkan/libVulkanSafeStruct.a
[0/1] Install the project...
-- Install configuration: "Release"
-- Up-to-date: /data/data/com.termux/files/usr/include/vulkan
-- Installing: /data/data/com.termux/files/usr/include/vulkan/layer
-- Installing: /data/data/com.termux/files/usr/include/vulkan/layer/vk_layer_settings.hpp
-- Installing: /data/data/com.termux/files/usr/include/vulkan/layer/vk_layer_settings.h
-- Installing: /data/data/com.termux/files/usr/include/vulkan/vk_enum_string_helper.h
-- Installing: /data/data/com.termux/files/usr/include/vulkan/utility
-- Installing: /data/data/com.termux/files/usr/include/vulkan/utility/vk_small_containers.hpp
-- Installing: /data/data/com.termux/files/usr/include/vulkan/utility/vk_sparse_range_map.hpp
-- Installing: /data/data/com.termux/files/usr/include/vulkan/utility/vk_safe_struct_utils.hpp
-- Installing: /data/data/com.termux/files/usr/include/vulkan/utility/vk_dispatch_table.h
-- Installing: /data/data/com.termux/files/usr/include/vulkan/utility/vk_concurrent_unordered_map.hpp
-- Installing: /data/data/com.termux/files/usr/include/vulkan/utility/vk_format_utils.h
-- Installing: /data/data/com.termux/files/usr/include/vulkan/utility/vk_struct_helper.hpp
-- Installing: /data/data/com.termux/files/usr/include/vulkan/utility/vk_safe_struct.hpp
-- Installing: /data/data/com.termux/files/usr/lib/libVulkanLayerSettings.a
-- Installing: /data/data/com.termux/files/usr/lib/libVulkanSafeStruct.a
-- Installing: /data/data/com.termux/files/usr/lib/cmake/VulkanUtilityLibraries/VulkanUtilityLibraries-targets.cmake
-- Installing: /data/data/com.termux/files/usr/lib/cmake/VulkanUtilityLibraries/VulkanUtilityLibraries-targets-release.cmake
-- Installing: /data/data/com.termux/files/usr/lib/cmake/VulkanUtilityLibraries/VulkanUtilityLibrariesConfig.cmake
INFO: READELF=llvm-readelf ... /home/builder/.termux-build/_cache/android-r26b-api-24-v4/bin/llvm-readelf
INFO: Generating symbols regex to /tmp/tmp.Uh9GQATJan
INFO: Done ... 0s
INFO: Total symbols 2339
INFO: Identifying files with nproc=4
INFO: Done ... 1s
INFO: Found 2 / 16 files
INFO: Running symbol checks on 2 files with nproc=4
INFO: Done ... 0s
INFO: Found files with undefined symbols
INFO: Showing result
SKIP: ./lib/libVulkanLayerSettings.a
SKIP: ./lib/libVulkanSafeStruct.a
INFO: Done ... 0s
INFO: Found 0 files with undefined symbols after exclusion
termux - build of 'vulkan-utility-libraries' done
Downloading https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v1.3.290.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 4593k    0 4593k    0     0  5792k      0 --:--:-- --:--:-- --:--:-- 5792k
100 5142k    0 5142k    0     0  5883k      0 --:--:-- --:--:-- --:--:-- 6784k
Applying patch: 0001-no-android.patch
Applying patch: 0002-getenv.patch
Applying patch: 0003-layers-utils-android_ndk_types.h.patch
-- Android: Targeting API '24' with architecture 'arm64', ABI 'arm64-v8a', and processor 'aarch64'
-- The CXX compiler identification is Clang 17.0.2
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/builder/.termux-build/_cache/android-r26b-api-24-v4/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Could NOT find robin_hood (missing: robin_hood_DIR)
-- Configuring done (0.4s)
-- Generating done (0.0s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_C_FLAGS
    CMAKE_USE_SYSTEM_LIBRARIES
    DOXYGEN_EXECUTABLE


-- Build files have been written to: /home/builder/.termux-build/vulkan-validation-layers/build
[1/170] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/error_location_helper.cpp.o
[2/170] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/error_message/error_location.cpp.o
[3/170] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/pnext_chain_extraction.cpp.o
[4/170] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/feature_requirements_helper.cpp.o
[5/170] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_object_types.cpp.o
[6/170] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/error_message/logging.cpp.o
[7/170] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/convert_utils.cpp.o
[8/170] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/hash_util.cpp.o
[9/170] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/image_layout_utils.cpp.o
[10/170] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/vk_layer_extension_utils.cpp.o
[11/170] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/ray_tracing_utils.cpp.o
[12/170] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_extension_helper.cpp.o
[13/170] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/vk_layer_utils.cpp.o
[14/170] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/vk_struct_compare.cpp.o
[15/170] Building CXX object layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/bindless_descriptor_pass.cpp.o
FAILED: layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/bindless_descriptor_pass.cpp.o 
/home/builder/.termux-build/_cache/android-r26b-api-24-v4/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r26b-api-24-v4 --sysroot=/home/builder/.termux-build/_cache/android-r26b-api-24-v4/sysroot -DVK_ENABLE_BETA_EXTENSIONS -DVK_USE_PLATFORM_WAYLAND_KHR -DVK_USE_PLATFORM_XCB_KHR -DVK_USE_PLATFORM_XLIB_KHR -DVK_USE_PLATFORM_XLIB_XRANDR_EXT -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/. -I/home/builder/.termux-build/vulkan-validation-layers/src/layers -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/. -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wpointer-arith -Wconversion -Wimplicit-fallthrough -Wstring-conversion -Wno-sign-conversion -Wno-implicit-int-conversion -MD -MT layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/bindless_descriptor_pass.cpp.o -MF layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/bindless_descriptor_pass.cpp.o.d -o layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/bindless_descriptor_pass.cpp.o -c /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/bindless_descriptor_pass.cpp
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/bindless_descriptor_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/./module.h:23:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/./type_manager.h:20:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:273:19: error: no member named 'OpExtInstWithForwardRefsKHR' in namespace 'spv'
        case spv::OpExtInstWithForwardRefsKHR:
             ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:284:19: error: no member named 'OpConstantCompositeReplicateEXT' in namespace 'spv'
        case spv::OpConstantCompositeReplicateEXT:
             ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:285:19: error: no member named 'OpSpecConstantCompositeReplicateEXT' in namespace 'spv'; did you mean 'OpSpecConstantCompositeContinuedINTEL'?
        case spv::OpSpecConstantCompositeReplicateEXT:
             ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                  OpSpecConstantCompositeContinuedINTEL
/data/data/com.termux/files/usr/include/spirv/unified1/spirv.hpp:2136:5: note: 'OpSpecConstantCompositeContinuedINTEL' declared here
    OpSpecConstantCompositeContinuedINTEL = 6092,
    ^
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/bindless_descriptor_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/./module.h:23:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/./type_manager.h:20:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:286:19: error: no member named 'OpCompositeConstructReplicateEXT' in namespace 'spv'
        case spv::OpCompositeConstructReplicateEXT:
             ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:647:19: error: no member named 'OpExtInstWithForwardRefsKHR' in namespace 'spv'
        case spv::OpExtInstWithForwardRefsKHR:
             ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:659:19: error: no member named 'OpConstantCompositeReplicateEXT' in namespace 'spv'
        case spv::OpConstantCompositeReplicateEXT:
             ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:660:19: error: no member named 'OpSpecConstantCompositeReplicateEXT' in namespace 'spv'; did you mean 'OpSpecConstantCompositeContinuedINTEL'?
        case spv::OpSpecConstantCompositeReplicateEXT:
             ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                  OpSpecConstantCompositeContinuedINTEL
/data/data/com.termux/files/usr/include/spirv/unified1/spirv.hpp:2136:5: note: 'OpSpecConstantCompositeContinuedINTEL' declared here
    OpSpecConstantCompositeContinuedINTEL = 6092,
    ^
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/bindless_descriptor_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/./module.h:23:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/./type_manager.h:20:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:661:19: error: no member named 'OpCompositeConstructReplicateEXT' in namespace 'spv'
        case spv::OpCompositeConstructReplicateEXT:
             ~~~~~^
8 errors generated.
[16/170] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vk_layer_config.cpp.o
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vk_layer_config.cpp:70:13: warning: unused function 'PropCallback' [-Wunused-function]
static void PropCallback(void *cookie, [[maybe_unused]] const char *name, const char *value, [[maybe_unused]] uint32_t serial) {
            ^
1 warning generated.
[17/170] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_function_pointers.cpp.o
[18/170] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_dispatch_table_helper.cpp.o
ninja: build stopped: subcommand failed.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-07-13 00:33:17 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/9915681926">9915681926</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/vulkan-validation-layers/build.sh
+++ b/packages/vulkan-validation-layers/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/KhronosGroup/Vulkan-ValidationLayers
 TERMUX_PKG_DESCRIPTION="Vulkan Validation Layers"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.3.289"
+TERMUX_PKG_VERSION="1.3.290"
 TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=4054b035879e90d6f0b7f993069fa50499c87776ae6c981c65f8a13ed69e12a1
+TERMUX_PKG_SHA256=37fd611f1c5c5879a68c6f63b09e600a3858f519aa147a2535961fa3d8e889ae
 TERMUX_PKG_DEPENDS="libc++, vulkan-loader"
 TERMUX_PKG_BUILD_DEPENDS="libwayland, libx11, libxcb, libxrandr, spirv-headers, spirv-tools, vulkan-headers (=${TERMUX_PKG_VERSION}), vulkan-utility-libraries (=${TERMUX_PKG_VERSION})"
 TERMUX_PKG_ANTI_BUILD_DEPENDS="vulkan-loader"

